routes/utils/request-to-bearer-token.js

```diff
@@ -1,8 +0,0 @@
-module.exports = toBearerToken
-
-function toBearerToken (request) {
-  if (!request.headers.authorization) {
-    return ''
-  }
-  return request.headers.authorization.substr(7)
```

Comment:
Fair point - based on this it's not major, as a user would still be able to see what's going on when the search for the word `session` in this repo, as the function contains `session` :)